test/adapters/sequelize.test.js

```diff
@@ -75,6 +75,7 @@ function getField(schema, name) {
             field: 'username',
             type: 'String',
             columnName: 'username',
+            isPrimaryKey: false,
```

Comment:
As you can see here the `username` field does not contain all the properties ; only `field`/`type`/`isPrimary` and not `isRequired`/`isFilterable`/`isVirtual`/...
It is not the concern of the adapter to aggregate the "default" values of all properties, I think it is currently done in `forest-express` dependency when the `.forestadmin-schema.json` content is serialized.

Reply:
Ok, I'll make sure that the property is always set by `forest-express`.